src/ed_core/application/features/order/handlers/queries/get_orders_query_handler.py

```
@@ -0,0 +1,38 @@
+                for order in self._uow.order_repository.get_all()
+            ]
+            return BaseResponse[list[OrderDto]].success(
+                "Orders fetched successfully.",
+                order_dtos,
+            )
+
+        except Exception as e:
+            raise ApplicationException(
+                Exceptions.InternalServerException, "Error fetching orders", [
+                    str(e)]
+            )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)